### PR TITLE
Editorial: improve a11y guidelines for share sheet

### LINKED ISSUE #264: Suggesting preview for security should mention accessibility
The security considerations for the [`share` method](https://www.w3.org/TR/2022/PR-web-share-20221215/#share-method) include the following suggestion:

> Content that is previewed and authorized by a user might be safe to forward, however it is not always possible for a person to identify when information should be confidential, so forwarding any content presents a risk. In particular, the [title](https://www.w3.org/TR/2022/PR-web-share-20221215/#dom-sharedata-title) might be used by an attacker to trick a user into misinterpreting the nature of the content.

As well as pointing out that `title` can be used to mislead, it would make sense to point to the accessibility considerations, and/or note that if such preview methods are not designed for high quality accessibility the proposed mitigation of security will fail.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #264

@chaals, please take a look. Any suggestions for improving this welcomed!

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/web-share/pull/270.html" title="Last updated on Jan 24, 2023, 1:10 AM UTC (48a94cc)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-share/270/783e54b...48a94cc.html" title="Last updated on Jan 24, 2023, 1:10 AM UTC (48a94cc)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -708,14 +708,39 @@ <h2>
...
       </h2>
       <p>
         When this specification is used to present information in the user
         interface, implementors will want to follow the OS level accessibility
-        guidelines for the platform.
+        guidelines for the platform. Further, as sharing can have potential
+        security implications for end-users, as outlined as part of the
+        {{Navigator/share()}} method, the share UI needs to be presented in an
+        accessible manner, while also taking into consideration a platform's
+        security guidelines for user interfaces:
```
Comment (chaals) [inline index.html:722]:
```suggestion
        security guidelines for user interfaces. Some key considerations are to:
```

File: index.html [hunk 2]
```diff
@@ -708,14 +708,39 @@ <h2>
...
+        type and/or nature of the content being shared.
+        </li>
+        <li>Ensure that users can navigate through the previewed content using
+        keyboard-only controls, and that the keyboard focus is clearly visible.
+        </li>
+        <li>Use headings and labels to ensure that the previewed content is
+        properly structured and can be understood by users using assistive
+        technology.
+        </li>
+        <li>And consider the color contrast, font size and type, and use of
```
Comment (chaals) [inline index.html:736]:
```suggestion
        <li>Ensure sufficient color contrast and font size, and use fonts,
```

File: index.html [hunk 3]
```diff
@@ -708,14 +708,39 @@ <h2>
...
+        icons or images presented in the share UI.
```
Comment (chaals) [inline index.html:737]:
```suggestion
        icons, and images that are widely recognised and easily understood, in the share UI.
```

File: index.html [hunk 4]
```diff
@@ -708,14 +708,39 @@ <h2>
...
+        </li>
+      </ul>
+      <p>
+        Together, these elements can help users with low vision or cognitive
```
Comment (chaals) [inline index.html:741]:
```suggestion
        Together, these elements can help users with a range of visual, motor or cognitive
```

**Review by chaals — APPROVED** [review]
some suggested editorial tweaks. Either way, this is a net improvement.

**Comment by marcoscaceres** [comment]
Thanks for you suggestions @chaals! Super helpful.

**Pull request merged**